src/app/lists/combo/combo-remote/combo-remote.component.html

```diff
@@ -10,6 +10,6 @@
     (closing)="onClosing()" (opened)="onOpened()"
     (closed)="onClosed()"
     placeholder="Location(s)"
-    [filterable]="true">
+    [filteringOptions]="{ filterable: true, caseSensitive: false }">
```

Comment:
These are just the default values, can be omitted entirely, same for the styling sample //CC @Lipata